easybuild/easyblocks/q/quantumespresso.py

```diff
@@ -361,28 +464,19 @@ def sanity_check_step(self):
         if 'yambo' in self.cfg['buildopts']:
             yambo_bins = ["a2y", "p2y", "yambo", "ypp"]
 
-        pref = self.install_subdir
+        d3q_bins = []
+        if 'd3q' in self.cfg['buildopts']:
+            d3q_bins = ['d3_asr3.x', 'd3_import3py.x', 'd3_lw.x', 'd3_q2r.x',
+                        'd3_qq2rr.x', 'd3q.x', 'd3_r2q.x', 'd3_recenter.x',
+                        'd3_sparse.x', 'd3_sqom.x', 'd3_tk.x']
 
         custom_paths = {
-                        'files': [os.path.join(pref, 'bin', x) for x in bins] +
-                                 [os.path.join(pref, 'upftools', x) for x in upftools] +
-                                 [os.path.join(pref, 'WANT', 'bin', x) for x in want_bins] +
-                                 [os.path.join(pref, 'YAMBO', 'bin', x) for x in yambo_bins],
-                        'dirs': [os.path.join(pref, 'include')]
+                        'files': [os.path.join('bin', x) for x in bins] +
+                                 [os.path.join('bin', x) for x in upftools] +
+                                 [os.path.join('bin', x) for x in want_bins] +
+                                 [os.path.join('bin', x) for x in yambo_bins] +
+                                 [os.path.join('bin', x) for x in d3q_bins],
```

Comment:
@akesandgren Since everything is located in the same directory now, you can trim this down (and why not fix the indent while you're at it):

```python
custom_paths = {
    'files': [os.path.join('bin', x) for x in bins + upftools + want_bins + yambo_bins + d3q_bins],
    'dirs': ['bin'],
}
```

Also, checking for a non-empty `bin` directory is kind of silly when we're also checking for a bunch of binaries in that dir, so might as well make `dirs` use `[]`.